go-controller/pkg/config/config.go

```diff
@@ -157,6 +157,14 @@ var (
 	OvnKubeNode = OvnKubeNodeConfig{
 		Mode: types.NodeModeFull,
 	}
+
+	// EgressIP holds egressIP parsed config file parameters and command-line overrides
+	EgressIPCfg = EgressIPConfig{
```

Comment:
I was thinking we just expose a single configuration knob "EgressIPReachabiltyTimeout" type uint and then we just retry at least 1 time per second, and no more than once every 100ms. If the user configures 0 it is disabled. I'm worried about providing this many configuration knobs to a user. @dcbw wdyt?

Reply:
so u are saying `EgressIPReachabiltyTimeout` is the timeout the user will provide in seconds for the initial dial timeout and if dial fails we use 1/10th of this value as a dial retry timeout. 
if 0 skip the dial check right ? hardcode number of retry to 2 as in osdn